osgpkitools/cert_utils.py

```diff
@@ -30,7 +30,7 @@ def get_ssl_context(usercert, userkey):
     pass_str = 'Please enter the pass phrase for'
     for _ in range(0, 2): # allow two password attempts
         def prompt_for_password(verify):
-            return getpass.getpass(pass_str+" '%s':" % userkey)
+            return getpass.getpass(pass_str+f" '{userkey}':")
```

Comment:
```suggestion
            return getpass.getpass(pass_str + f" '{userkey}':")
```

Just a quick whitespace change for my own sanity